data/css/modules/card/_default.scss

```diff
@@ -209,6 +209,19 @@
                 padding: 20px;
             }
         }
+
+        // Video/audio icon sizing
```

Comment:
I don't see this in the specs that you linked in the Phabricator issue; the icons are all large-sized on the polaroid samples shown there. Even so, shouldn't it be `&.height-b, &.height-a` without regard to width, as in the post card below?